python/tvm/relay/grammar/Relay.g4

```diff
@@ -19,7 +19,7 @@
 
 grammar Relay;
 
-SEMVER: 'v0.0.1' ;
+SEMVER: 'v0.0.1.1' ;
```

Comment:
```suggestion
SEMVER: 'v0.0.2' ;
```

This is the convention for pre-release semver.

Reply:
Thanks, done